src/app/components/colors/color_hex_input.tsx

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as React from "react";
+import { colorFromHTMLColor, colorToHTMLColorHEX } from "../../../core";
+import { ColorSpacePickerState, InputField } from "../color_space_picker";
+
+interface ColorHexInputProps {
+  state: ColorSpacePickerState;
+  updateState: (state: ColorSpacePickerState) => void;
+}
+
+export class ColorHexInput extends React.Component<
+  ColorHexInputProps,
+  Record<string, unknown>
+> {
+  render() {
+    const currentColor = this.props.state.desc.toRGB(
+      this.props.state.x1,
+      this.props.state.x2,
+      this.props.state.x3
+    );
+    const rgb = { r: currentColor[0], g: currentColor[1], b: currentColor[2] };
+
+    return (
+      <>
```

Comment:
Same here, missing a tag.

Reply:
This is short syntax of react fragment. 
https://reactjs.org/docs/fragments.html